src/Microsoft.ML.Transforms/KeyToVectorMapping.cs

```diff
@@ -456,16 +424,18 @@ private void EncodeValueToBinary(BufferBuilder<float> bldr, uint value, int bits
         }
     }
 
+    /// <summary>
+    ///  Converts the key types back to binary verctors.
```

Comment:
>verctors [](start = 47, length = 8)

vectors

Reply:
Thanks for catching!

---
In reply to: [253717192](https://github.com/dotnet/machinelearning/pull/2367#discussion_r253717192) [](ancestors = 253717192)